api-repository.md

```diff
@@ -28,13 +28,36 @@ Full-blown frameworks that operate on top of Bukkit and define how you write you
 
   > kelp is an all-in-one framework that aims to avoid version-depended code.
 
-  __Main features__:
+  __Core features__:
```

Comment:
You renamed `Main` to `Core`. Should this be adopted for all occurrences of `Main`?